source/_integrations/subaru.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: Subaru 
+description: Instructions on how to integrate Tesla car into Home Assistant.
+ha_category:
+  - Car
+  - Binary Sensor
+  - Climate
+  - Presence Detection
+  - Lock
+  - Sensor
+  - Switch
+ha_release: 0.111
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@G-Two'
+ha_domain: subaru
+---
+
+The `Subaru` integration offers integration with the [Subaru](https://www.mysubuaru.com) cloud service and provides presence detection as well as sensors such as charger state and temperature.
+
+This integration provides the following platforms:
+
+- Binary sensors - EV charger connection status.
+- Sensors - such as Outside temperature, odometer, estimated range, and EV information (battery level, range, charging rate).
+- Device tracker - obtain location of your car.
+- Lock - Lock and Unlock Doors.
+- Climate - Enables control of HVAC and Remote Engine Start (turn on/off, set target temperature).
+- Switch - Remote actuate Horn, Lights, request Location, and start EV charging.
+
+## Configuration
+
+### Home Assistant UI
+  * The Subaru integration offers configuration through the Home Assistant UI:
+    
+    **Configuration** -> **Integrations** -> **Add** -> **Subaru**
+
+### YAML
```

Comment:
Since this is a new integration, that interacts with a device or service, it should not supply YAML based configurations.

See ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>

Reply:
YAML references removed.  Thanks for pointing me towards the ADRs.